install.sh

```diff
@@ -707,14 +717,18 @@ openrc_enable() {
 # Start openrc service
 openrc_start() {
   info "openrc: Starting node_exporter"
-  $SUDO $FILE_NODE_EXPORTER_SERVICE restart
+  $SUDO "$FILE_NODE_EXPORTER_SERVICE" restart
 }
 
 # relabel to executable if SElinux is installed
-selinux() {
-  if getenforce > /dev/null; then
-     semanage fcontext -D $BIN_DIR/node_exporter
-     semanage fcontext -a -t bin_t $BIN_DIR/node_exporter && restorecon -v $BIN_DIR/node_exporter
+setup_selinux() {
+  if type -p getenforce > /dev/null 2>&1; then
+    if type -p semanage > /dev/null 2>&1; then
+       semanage fcontext -D "$BIN_DIR/node_exporter"
+       semanage fcontext -a -t bin_t "$BIN_DIR/node_exporter" && restorecon -v "$BIN_DIR/node_exporter"
+    else
+       info "Cannot setup SELinux context for binary \"$BIN_DIR/node_exporter\". Please install policycoreutils-python package."
```

Comment:
Change to:
_"Cannot setup SELinux context for binary '$BIN_DIR/node_exporter'. Please install 'policycoreutils-python' package"_

Reply:
Fixed